src/plugins/expressions/common/expression_functions/specs/math_column.ts

```diff
@@ -52,6 +53,14 @@ export const mathColumn: ExpressionFunctionDefinition<
       }),
       required: true,
     },
+    castColumns: {
```

Comment:
Shouldn't we cast all the cells by default instead of introducing an additional argument?

Reply:
It's pretty hard to tell which columns are actually used by the tinymath expression. Casting all of them will most likely cause a lot of unnecessary work because most columns won't be referenced in the expression.